.github/workflows/rust.yml

```diff
@@ -28,8 +28,8 @@ env:
   CARGO_TERM_COLOR: always
   CICD_INTERMEDIATES_DIR: "_cicd-intermediates"
   XDG_CACHE_HOME: ${{ github.workspace }}/.cache
-  SCCACHE_GHA_ENABLED: "true"
-  RUSTC_WRAPPER: "sccache"
+  # SCCACHE_GHA_ENABLED: "true"
```

Comment:
do you think we need to remove them entirely or you want to add later a comment why they are commented out?

Reply:
Lets just remove them